test/config/pretest.js

```diff
@@ -3,7 +3,7 @@
 
 var chaiHttp = require("chai-http");
 const { MongoClient } = require("mongodb");
-const uri = "mongodb://localhost:27017/scicat";
+const uri = process.env.MONGODB_URI;
```

Comment:
If we set the test DB URI as the MONGODB_URI, it will wipe the local development DB if someone forgets to change the URI before running the test. Therefore, I suggest keeping it unchanged for now.

Reply:
Good point. `appUrl` is also hard-coded a few lines down. Unless there was a particular reason this needs to be changed at runtime?

Reply:
I don't think we need `appUrl` to be changed at run time. I'm not sure if we have environment variables to assign to the `appUrl`

Reply:
After discussing with @nitrosx regarding the test database, he suggested using the MONGODB_URI variable for the test database as well. In the documentation, we can mention that one should change the MONGODB_URI before running the test command to prevent accidental local data wipes.

Here are some minor suggestions for the code changes:

- Include require("dotenv").config() to load the environment variables. Without this, the environment variables will be undefined.
- For global.db, replace client.db(uri.split("/").pop()); with client.db().

```suggestion
require("dotenv").config();
var chaiHttp = require("chai-http");

const { MongoClient } = require("mongodb");

const client = new MongoClient(process.env.MONGODB_URI);

async function loadChai() {
  const { chai } = import("chai");
  chai.use(chaiHttp);
  await client.connect();
}

loadChai();
global.appUrl = "http://localhost:3000";
global.request = require("supertest");
global.db = client.db();
```